frontend/integration-tests/views/dashboard-shared.view.ts

```diff
@@ -0,0 +1,5 @@
+import { browser, ExpectedConditions as until } from 'protractor';
+import { untilNoLoadersPresent } from '../views/crud.view';
+
+const extraWaitingLime = 2000;
```

Comment:
Typo (Lime/Time), and 1000 should be sufficient.

Reply:
a small nitpick: 
- `UPPER_CASE` makes constants more distinguishable 
- could be better named (useful mainly when adding other constants to this file)

Reply:
I sometimes got "StaleElementReferenceError: stale element reference: element is not attached to the page document" until I increased the waiting time

Reply:
> a small nitpick:
> 
> * `UPPER_CASE` makes constants more distinguishable
> * could be better named (useful mainly when adding other constants to this file)

Fixed